addons/ingress-nginx-metrics/README.md

```diff
@@ -0,0 +1,28 @@
+
+## Deployment
+### AWS
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/ingress-nginx/v1.6.0.yaml
+```
+
+### GCE
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/ingress-nginx/v1.6.0-gce.yaml
+```
```

Comment:
Should we change this to point to this repo?

Reply:
Changed in 0d0a4f0b5.